content/_redirects

```diff
@@ -39,6 +39,7 @@
 /docs/alerting/             /docs/alerting/latest/overview/             302!
 /docs/practices/            /docs/practices/naming/                     302!
 /docs/guides/               /docs/guides/basic-auth/                    302!
+/docs/*                     /docs/404/                                  302!
```

Comment:
I had a read through, and seems to me this is the way to do a redirect. But have not been able to test it on my local, so I'd be grateful for pointers on how to do it.